package.json

```diff
@@ -18,15 +19,16 @@
     "yup": "^0.32.9"
   },
   "scripts": {
+    "start": "react-scripts start",
     "build": "react-scripts build",
     "eject": "react-scripts eject",
-    "format": "prettier --write .",
-    "formatHusky": "prettier --write",
```

Comment:
I think It is better to have separated scripts, one for CI and one for Husky (local), or I am wrong?

Reply:
23 formats everything in the working dir, is that the expected behavior? imo it's better to add the format pattern, and merge these two into one single command